### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by iZakirSheikh** [mahozad/wavy-slider#12]
Instead of changing API daily you should thoroughly think it and then publish.
Maybe utilize **ChatGPT**, **Gemini** for brainstorming ideas.

**Comment by mahozad** [comment]
You're right. Thanks for the suggestions and sorry for the constant change.

I think the library is now "complete" and the current API is going to be the final shape for version `1.0.0`.
There will probably be just inner optimizations and refactorings for the stable `1.0.0` release.

PS: Have you used the library on iOS? Does it work as expected?

**Comment by iZakirSheikh** [comment]
I have used the library in my android app called [Audiofy](https://play.google.com/store/apps/details?id=com.prime.player)
The app uses version 0.8.0 of ur library.
I think the things that currently Library misses include:

1. Thumb making.
2. Loading progress
3. Provide a way to display chapters like YouTube.
4.

**Comment by iZakirSheikh** [comment]
Actually I am on mobile.
My app currently uses material2. So I can't confirm material3 implementation. 
1. By loading progress I mean the slider should show buffered upto. I.e., how much content have loaded.
Since this is actually meant to show progress of media, so there should be a way to show stepped progress, chapters etc. we have to brainstorm maybe using figma before building it further.

Sorry for my bad English, actually I am not a native speaker.
